src/web/test_sws.erl

```diff
@@ -0,0 +1,36 @@
+-module(test_sws).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+-export([start/0]).
+
+
+start() ->
+    io:format("start call ~n",[]),
+    sws:start(fun basic_handle/2, 8045).
+
+basic_handle(_S, ReqList) ->
+    io:format("handle call ~n",[]),
+    lists:foreach(
+      fun(E) -> 
+	      io:format("~p~n",[E])
+      end, 
+      ReqList),
+    {200, [], "<pre>\n" ++ io_lib:format("~p~n~n Time: ~p~n~n System Info: ~p~n", [ReqList, erlang:localtime(), os:cmd("uname -a")]) ++ "</pre>\n"}.
+
+
+ 
+basic_handle_test()->
+    basic_handle(none, 
```

Comment:
um, is it even possible for this test to fail?

Reply:
haha, the code kept failing, so that's why i wrote this test :scream_cat: 

this way I didn't have to start a server to see the error, it would just come up during the unit tests

Reply:
so, more of a debug thing